charts/cloudquery/values.yaml

```diff
@@ -75,7 +75,7 @@ config: |
   spec:
     name: aws
     path: cloudquery/aws
-    version: "v7.3.2" # latest version of aws plugin
+    version: "v7.4.0" # latest version of aws plugin
```

Comment:
@disq I'm not sure, doesn't it need the chart version bump as well?
See https://github.com/cloudquery/helm-charts/actions/runs/3684455929/jobs/6234212033

Reply:
Yes it does as it's part of the default configuration of the chart. Also no need to review these PRs they get automerged by our bots